projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.html

```diff
@@ -1,32 +1,41 @@
-<ng-container *ngIf="cart$ | async as cart">
+<ng-container
+  *ngIf="{
```

Comment:
Hoisted `cart$` and `isCartTransferAllowedForSeller$` so they can be used throughout the template.